Utilities/bootstrap

```diff
@@ -41,6 +41,7 @@ import shutil
 import subprocess
 import sys
 import tempfile
+import codecs
```

Comment:
nit-pick: Let's keep this like in alphabetical order. How about putting this between `import argparse` and `import errno`?